app/src/main/java/com/example/molliestephenson/tictactoe/MainActivity.java

```diff
@@ -48,13 +51,25 @@ private void createBoard() {
         mBoard.add(mRightBottom);
     }
 
-    private void createGame() {
-        Board board = new Board(3);
-        Player player1 = new MobilePlayer(Marks.X);
-        Player player2 = new MobilePlayer(Marks.O);
+    private void createGame(boolean isAHumanVHumanGame, boolean isA3x3Board) {
```

Comment:
It shouldn't be possible for that to happen.  If no buttons are pressed in the menu, so that no extras are added to the intent, the default values are set when attempting to get the data from the intent and I have set the default values to true.  I am going to change to boolean values to Player types, which should make things easier when adding game types.  I was really just trying to get some form of data passed between activities. 

 In regards to the above comment, I have an enum from the core package that I will use, but since I was not actually using the extras passed with the Intent in any meaningful way, I wasn't worrying too much about it. Thanks for feedback!